sky/engine/bindings/dart_runtime_hooks.cc

```diff
@@ -37,6 +39,14 @@ extern void syslog(int, const char *, ...);
 }
 #endif
 
+namespace dart {
+namespace bin {
+
+extern bool capture_stdout;
```

Comment:
We should create a header file that declares these symbols so we can see the dependencies in the codebase by looking at the include rules.